src/tests/utils_test.ts

```diff
@@ -1,98 +1,376 @@
-import { AxiosResponse } from "axios";
-import { InvalidUnsignedPayload } from "../coinbase/errors";
-import { logApiResponse, parseUnsignedPayload } from "./../coinbase/utils"; // Adjust the path as necessary
-
-describe("parseUnsignedPayload", () => {
-  it("should parse and return a valid payload", () => {
-    const payload = "7b226b6579223a2276616c7565227d"; // {"key":"value"} in hex
-    const expectedOutput = { key: "value" };
-    expect(parseUnsignedPayload(payload)).toEqual(expectedOutput);
-  });
+import * as crypto from "crypto";
+import { InternalError } from "../coinbase/errors";
+import {
+  AddressBalanceList,
+  AddressList,
+  Address as AddressModel,
+  Balance as BalanceModel,
+  User as UserModel,
+  Wallet as WalletModel,
+} from "./../client/api";
+import { Coinbase } from "./../coinbase";
+import { WalletData } from "./../coinbase/types";
+import { User } from "./../coinbase/user";
+import { Wallet } from "./../coinbase/wallet";
+import {
+  VALID_ADDRESS_MODEL,
+  VALID_WALLET_MODEL,
+  addressesApiMock,
+  generateRandomHash,
+  generateWalletFromSeed,
+  mockReturnRejectedValue,
+  mockReturnValue,
+  newAddressModel,
+  walletsApiMock,
+} from "./utils";
+import Decimal from "decimal.js";
+import { FaucetTransaction } from "../coinbase/faucet_transaction";
 
-  it("should throw InvalidUnsignedPayload error if payload cannot be parsed", () => {
-    const payload = "invalidhexstring";
-    expect(() => parseUnsignedPayload(payload)).toThrow(InvalidUnsignedPayload);
-  });
+describe("User Class", () => {
```

Comment:
@marcin-cb I think it is user_test.ts

Reply:
yeah fixed this, did a wrong copy and paste.